LibraBFT/Impl/Consensus/EpochManagerTypes.agda

```diff
@@ -0,0 +1,208 @@
+{- Byzantine Fault Tolerant Consensus Verification in Agda, version 0.9.
```

Comment:
OK.
BTW: that is generally true for all our types files : not in same order as Haskell.
Because:
- haskell order dictated by template haskell
- agda order dictated by define before use